docs/api/process.md

```diff
@@ -98,6 +98,12 @@ The `process` object has the following methods:
 
 Causes the main thread of the current process crash.
 
+### `process.getCreationTime()`
+
+Returns [`number`]
+Indicates the creation time of the application.
+The time is represented as microseconds (s/1,000,000) since the Windows epoch (1601-01-01 00:00:00 UTC)
```

Comment:
mac & linux seem to return seconds from the start of the process.
And can be null or empty if the API fails.

Reply:
Looking at the run from Mac it is giving back time since windows epoch as well 
Here is what we get when run on a Mac.
131750664685357260
2018-07-03T04:41:08.535Z

Comment from time.h in chromium

/ Time represents an absolute point in coordinated universal time (UTC),
// internally represented as microseconds (s/1,000,000) since the Windows epoch
// (1601-01-01 00:00:00 UTC).

Reply:
> microseconds since the Windows epoch

The returned value should be easy to use in JavaScript.
JavaScript [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object uses "number of milliseconds since 1 January 1970 UTC" (the epoch).

Reply:
> And can be null or empty if the API fails.

Please return `null` if the time cannot be obtained.